desktop-widgets/tab-widgets/TabDiveEquipment.cpp

```
@@ -81,7 +81,8 @@ TabDiveEquipment::TabDiveEquipment(QWidget *parent) : TabBase(parent),
 	for (int i = 0; i < CylindersModel::COLUMNS; i++) {
 		if (ignoreHiddenFlag(i))
 			continue;
-		bool checked = s.value(QString("column%1_hidden").arg(i)).toBool();
+		auto setting = s.value(QString("column%1_hidden").arg(i));
+		bool checked = !setting.isValid() && i == CylindersModel::SENSORS ? true : setting.toBool();
```

So played around with hiding the sensors by default. While doing this I noticed that the depth (deco switch at) column used to be hidden by default, but isn't anymore. Not sure what is best long term. It's easy to discover features if they're visible by default and you can hide them if you don't need them. At the same time, there is no point in showing things by default that isn't useful for most users and with every new column it starts getting crowded.

---

Careful: The CylindersModel code is a complete disaster, because it is used in two rather different places: in the planner and the equipment tab.

I could imagine that the column should not be shown in the equipment tab at all.

---

This code didn't seem to be used in the planner and the column was always visible there, so I made it always hidden in the planner.

Still for the non-planner case, is this how we want to have it? Keep it hidden by default and people who might need it can discover it later? If so, should i make it more generic and possibly hide some other column by default?

---

Decided to keep it hidden by default since I didn't get a reply to this. Also made it a bit more generic so it's easier to add new column as hidden by default if that need comes up.